readme.md

```diff
@@ -56,11 +56,26 @@ grunt.initConfig({
 });
 ```
 
+### Base config
+
+```js
+grunt.initConfig({
+	eslint: {
+		options: {
+			baseConfig: {
+				extends: ["eslint-config-shared"]
```

Comment:
single-quotes

Reply:
Would prefer `'plugin:ava/recommended'` here?

Reply:
Nah, it's just an example.